tests/foreman/api/test_provisioningtemplate.py

```diff
@@ -751,3 +751,73 @@ def test_positive_verify_chronyd_timesource_kickstart_template(
             assert f'timezone --utc {timezone}' in render
         else:
             assert f'timezone --utc {timezone} --ntpservers {ntp_server_value}' in render
+
+    @pytest.mark.rhel_ver_match(r'^(?!.*fips).*$')
```

Comment:
You won't need this, This is used when we use `module_sync_kickstart_content` for enabling and syncing specific repos